ocw/lib/EC2.py

```diff
@@ -49,8 +49,11 @@ def check_credentials(self):
                 self.get_all_regions()
                 return True
             except Exception:
-                self.log_info("check_credentials (attemp:{}) with key {} expiring at {} ", i, self.__key,
-                              self.__credentials.getAuthExpire())
+                self.loggerAdapter.info(
```

Comment:
```
self.loggerAdapter.info("check_credentials (attemp:{}) with key {} expiring at {} ", i, self.__key, self.__credentials.getAuthExpire()))
```

in  such a case use of format is not necessary . Logger functions may handle formatting internally . This comments applies to all other lines where you are using format within debug/info/warning/error

Reply:
please ignore this